python/pyspark/pandas/series.py

```diff
@@ -147,6 +147,7 @@
 Parameters
 ----------
 other : Series or scalar value
+fill_value : Scalar value, default None
```

Comment:
FYI: This documentation is a bit different from [pandas](https://pandas.pydata.org/docs/reference/api/pandas.Series.add.html), since we currently doesn't support " If data in both corresponding Series locations is missing the result of filling (at that location) will be missing.", which requires combine two frame that is potentially overkill the performance in pandas API on Spark